src/protagonist/API/Features/Image/Validation/ImageBatchPatchValidator.cs

```diff
@@ -36,6 +36,7 @@ public ImageBatchPatchValidator(IOptions<ApiSettings> apiSettings)
             members.RuleFor(a => a.ImageOptimisationPolicy).Empty().WithMessage("Image optimisation policies cannot be set in a bulk patching operation");
             members.RuleFor(a => a.MaxUnauthorised).Empty().WithMessage("MaxUnauthorised cannot be set in a bulk patching operation");
             members.RuleFor(a => a.DeliveryChannels).Empty().WithMessage("Delivery channels cannot be set in a bulk patching operation");
+            members.RuleFor(a => a.ThumbnailPolicy).Empty().WithMessage("Thumbnail policy cannot be set in a bulk patching operation");
```

Comment:
is it worth adding a test to the validators for this?

Reply:
I forgot to mention in the summary of this PR that that one was added:
https://github.com/dlcs/protagonist/blob/aedab5e403cee931ece7d642bac65a847f6ae06d/src/protagonist/API.Tests/Features/Images/Validation/ImageBatchPatchValidatorTests.cs#L147C5-L147C5